xonsh/built_ins.py

```diff
@@ -720,6 +724,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_execer__ = execer
     builtins.__xonsh_commands_cache__ = CommandsCache()
     builtins.__xonsh_all_jobs__ = {}
+    builtins.__xonsh_enabled_builtin_commands__ = set(all_builtin_commands.keys())
```

Comment:
That said, xoreutils might still be fine.  I think you're right about builtins meaning a whole bunch of different things right now.